server/server/views/add_event.py

```diff
@@ -0,0 +1,83 @@
+"""View for adding new event to db"""
+from datetime import datetime
+
+from cornice import Service
+from cornice.validators import colander_body_validator
+from pyramid.security import Allow, Authenticated
+
+from ..models import model_to_dict
+from ..models.event import Event
+from ..models.category import Category
+from ..models.event_status import EventStatus
+from ..models.event_history import EventHistory
+from ..validation_schema import EventSchema
+
+
+class AddEventFactory(object):
+    """Factory for adding ACL to the Service"""
+
+    def __init__(self, request, context=None):
+        self.request = request
+
+    def __acl__(self):
+        return [(Allow, Authenticated, 'add')]
+
+
+add_event = Service(name='add_event', path='/add-event',
```

Comment:
Why don't you use cornice's [Resource](http://cornice.readthedocs.io/en/latest/resources.html) here? Event is more like resource than service.